src/resolvers/user-resolver.ts

```diff
@@ -24,11 +30,16 @@ export class UserResolver {
 
   @Query(() => [UserModel])
   async getUsers() {
-    return this.users.find();
+    return await this.users.find();
   }
 
   @Mutation(() => UserModel)
   async createUser(@Arg('userData') userData: CreateUserInput): Promise<UserModel> {
-    return this.users.save(userData);
+    const userExists = await this.users.find({ where: { email: userData.email } });
+
+    if (!userExists.length) {
+      return await this.users.save(userData);
```

Comment:
não precisa do await
```suggestion
      return this.users.save(userData);
```